src/use-built-ins-plugin.js

```diff
@@ -4,11 +4,11 @@ function isPolyfillSource(value) {
   return value === "babel-polyfill";
 }
 
-function warnOnInstanceMethod() {
-  // state.opts.debug &&
-  //   console.warn(
-  //     `Adding a polyfill: An instance method may have been used: ${details}`,
-  //   );
+function warnOnInstanceMethod(state, details) {
+  state.opts.debug &&
```

Comment:
Right now there's too many warnings being created when testing so not sure when to add this in.